src/Controller/EventsController.php

```diff
@@ -8,6 +8,7 @@
 use Drupal\Core\Datetime\DrupalDateTime;
 use Drupal\Core\Extension\ModuleHandlerInterface;
 use Drupal\openy_gated_content\VirtualYAccessTrait;
+use Drupal\taxonomy\Entity\Term;
```

Comment:
We need to add dependency to taxonomy module

https://github.com/froboy/yusaopeny_gated_content/blob/event-series-instructor/openy_gated_content.info.yml

Reply:
This `use` was outdated and was replaced with the entity queries. I've removed it instead.